### ISSUE TITLE
Invalid window  id: 1000

### ISSUE THREAD

**Issue by fgheng** [kyazdani42/nvim-tree.lua#1228]
### Description

Something went wrong while I close the window of nvim-tree.

![image](https://user-images.githubusercontent.com/50551377/166262247-540389e0-15f0-4c62-8a59-1012881a40d9.png)

### Neovim version

```markdown
NVIM v0.8.0-dev+95-g07660193a
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

### Operating system and version

arch linux

### nvim-tree version

483f155

### Steps to reproduce

1. open a file
2. open nvimtree
3. move cursor to the file and close it, now there is only nvimtree
4. open a file by nvimtree
5. move cursor to nvimtree
6. try to close it, error occured

### Expected behavior

close the window of nvim-tree with no error

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "kyazdani42/nvim-tree.lua",
      "kyazdani42/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

**Comment by alex-courtis** [comment]
Replicated.

**Comment by kyazdani42** [comment]
closed in #1235
